components/num-util/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "icu-num-util"
+description = "Low-level tools for dealing with numbers"
+version = "0.0.1"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "../../LICENSE"
+categories = ["internationalization"]
+include = [
+    "src/**/*",
+    "Cargo.toml",
+    "README.md"
+]
+
+[dependencies]
+smallvec = "1.4.1"
+static_assertions = "1.1.0"
+more-asserts = "0.2.1"
```

Comment:
I prefer being explicit about assumptions in code, even benign ones like "usize is at least as big as u16".  I would like to keep `static_assertions` for this purpose.  This is an assertion for the compiler: ICU4X should refuse to compile if that condition is not met.  It blows my mind that static assertions aren't in the standard library like they are in C++, where I use them frequently.  If static assertions ever make it there, it will be an easy change to make without needing to understand the intricacies of FixedDecimal.

I don't feel as strongly about `more-asserts`.  I added them because the error message is nicer if the assertions were to ever fail.  But, that's probably not necessary, because these assertions shouldn't ever fail.  I'll remove `more-asserts`.

The assertion crates are big winners in the size/benefit tradeoff, since they are not intended to add any runtime bloat.  I hear what you're saying about compile time though.  I haven't measured the degree to which that is affected.